compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/backhandler/BackEventCompat.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.backhandler
+
+import androidx.annotation.FloatRange
+import androidx.annotation.IntRange
+import androidx.compose.ui.ExperimentalComposeUiApi
+
+/**
+ * Object used to report back gesture progress.
+ * Holds information about the touch event, swipe direction and the animation progress
+ * that predictive back animations should seek to.
+ */
+@ExperimentalComposeUiApi
+class BackEventCompat(
```

Comment:
Isn't it supposed to be iOS-only for now? If we're talking about multiplatform support - I'd think once again if it's need to be done in this module.